configure

```diff
@@ -152,7 +151,7 @@ while : ; do
     -lib*)
         cclibs="$2 $cclibs"; shift;;
     -no-curses|--no-curses)
-        with_curses=no;;
+        ;;  # Ignored for backward compatibility
```

Comment:
A warning that this has no effet could be `echo`ed.

Reply:
I just followed the pattern of the `-with-pthread*` ignored option.